Doc/library/sqlite3.rst

```diff
@@ -73,27 +73,26 @@ This example uses the iterator form::
 
 .. _sqlite3-placeholders:
 
-Usually your SQL operations will need to use values from Python variables.  You
-shouldn't assemble your query using Python's string operations because doing so
-is insecure; it makes your program vulnerable to an SQL injection attack
-(see the `xkcd webcomic <https://xkcd.com/327/>`_ for a humorous example of
-what can go wrong)::
+SQL operations will usually need to use values from Python variables. It is not
+adviced to assemble queries using Python's string operations, because they are
```

Comment:
```suggestion
advised to assemble queries using Python's string operations, because they are
```
But something stronger might be good:
```suggestion
good practice to assemble queries using Python's string operations, because they are
```

Reply:
Thanks, @jacobtylerwalls! I've applied your suggestions.